test/logger_test.rb

```diff
@@ -0,0 +1,85 @@
+require 'test_helper'
+
+describe Lotus::Logger do
+  class IoCapture
+    attr_accessor :stream
+    def write(smt)
+      @stream = smt
+    end
+    def close;end
+    def flush;end
+  end
+
+  #alias STDOUT to custom class to test output
+  STDOUT = IoCapture.new
+
+  before do
+    #clear defined class
+    Object.constants.include?(:TestLogger)
+  end
+
+  it 'like std logger, default log level is info' do
+    class TestLogger1 < Lotus::Logger; end
+    assert_respond_to(TestLogger1.new, :info?)
+  end
+
+  it 'always use STDOUT' do
+    class TestLogger2 < Lotus::Logger; end
+    logger = TestLogger2.new
+    logger.info('foo')
+    assert_match(/foo/, STDOUT.stream)
+  end
+
+  describe '.app_tag' do
```

Comment:
the convention for `.` is to describe class method, in this case, maybe just `app_tag` to denote the fact that we are describing a feature of a function rather than instance/class method